packages/apollo-language-server/src/config/__tests__/config.ts

```diff
@@ -1,11 +1,29 @@
-import { ApolloConfig, ApolloConfigFormat } from "../";
+import {
+  ApolloConfig,
+  ApolloConfigFormat,
+  getGraphInfo,
+  loadConfigWithDefaults
+} from "../";
 import URI from "vscode-uri";
+import { DeepPartial } from "apollo-env";
+
+export function createConfig(raw: DeepPartial<ApolloConfigFormat>) {
```

Comment:
Q: it feels strange that we now have to perform this when creating configs for tests... would it make sense to try to move this transformation into the `ApolloConfig` construction / initialization?